hopscotch-web/src/main/java/charliek/hopscotch/web/ApiHandlerDecorator.java

```diff
@@ -8,8 +8,8 @@
 public class ApiHandlerDecorator implements HandlerDecorator {
 	@Override
 	public Handler decorate(Registry serverRegistry, Handler rest) throws Exception {
-		return Handlers.chain(serverRegistry, c -> {
+		return Handlers.chain(Handlers.chain(serverRegistry, c -> {
 			c.get("api/timed", TimedHandler.class);
-		});
+		}), rest);
```

Comment:
Thanks for taking a look. I debated on deleting the forum post, but I figured it was just noise and was hoping that maybe nobody spent time on it yet.  I'm slowly getting the hang of it.